.github/workflows/ci.yml

```diff
@@ -174,3 +192,64 @@ jobs:
           git diff -I'^"POT-Creation-Date: ' --numstat **/*.po **/*.pot | cut -f3 | xargs -r git add
           git add $(git ls-files -o --exclude-standard *.po *.pot) .tx/config
           git diff-index --cached --quiet HEAD || { git commit -m "Update translations from ${{ env.BRANCH }}" && git push; }
+
+
+  propagate_translations:
+    runs-on: ubuntu-latest
+    needs: [trigger, update]
+    permissions:
+      contents: read
+    strategy:
+      max-parallel: 4
+      fail-fast: false
+      matrix:
+        branch: ${{fromJson(needs.trigger.outputs.oldversions)}}
+        language: ${{fromJson(needs.trigger.outputs.languages)}}
+
+    steps:
+      - name: Checkout main
+        uses: actions/checkout@v3.5.3
+        
+      - name: Checkout branch ${{ needs.trigger.outputs.current }}
+        uses: actions/checkout@v3.5.3
+        with:
+          ref: ${{ needs.trigger.outputs.current }}
+          path: ${{ needs.trigger.outputs.current }}
+    
+      - name: Checkout branch ${{ matrix.branch }}
+        uses: actions/checkout@v3.5.3
+        with:
+          ref: ${{ matrix.branch }}
+          path: ${{ matrix.branch }}
+      
+      - name: Set up Python 3
+        uses: actions/setup-python@v4.7.0
+        with:
+          python-version: '3'
+          cache: 'pip'
+          cache-dependency-path: |
+            requirements.txt
+    
+      - name: Install dependencies
+        run: |
+          python3 -m pip install --upgrade pip
+          python3 -m pip install pomerge
+
+      - name: Merge ${{ matrix.language }} translations from ${{ needs.trigger.outputs.current }} ...
+        run: | 
+          cd ${{ needs.trigger.outputs.current }}/${{ matrix.language }}
+          pomerge --from-files *.po **/*.po
+      
+      - name: ... into ${{ matrix.branch }}
+        run: |
+          cd ${{ matrix.branch }}/${{ matrix.language }}
+          pomerge --to-files *.po **/*.po
```

Comment:
As you probably saw, this line fails with `OSError: Syntax error in po file (line 1)`.

I don't have an idea why 🤔 Some debug print would probably help.

I'll try to play with pomerge locally soon, maybe would be able to reproduce it locally?

Reply:
I'm assuming it failed because I was assuming there was po files in language dir  but there is anothee subdirectory "LC_MESSAGES".

pl/LC_MESSAGES/*.po is correct, pl/*.po is not

Reply:
Hm, wouldn't `**/*.po` include also `pl/LC_MESSAGES/` 🤔 

🤞

Reply:
Pomerge is might be sensible. **/*.po should work, but *.po does not, so the whole run failed.

Reply:
Yep, fixed. Had tp enable globstar, as mentioned in that weblate script's PR